### PR TITLE
announcing Lucas video on youtube

### PR DESCRIPTION
**Pull request by samumantha**
please feel free to edit directly if you have better idea for text ( I do not believe that I am the right person for this job) :) 
As usual this is more to initiailize the posting 

Also, I started adding dates to the tweet filenames to not have to come up with new unique text for every tweet

### CODE REVIEW

File: tweets/210916_video.tweet [hunk 1]
```diff
@@ -0,0 +1,3 @@
+For everyone who missed or wants to revisit Lucas talk about #JuliaLang package development of our #ResearchSoftwareSeminarSeries:
```
Comment (lucaferranti) [inline tweets/210916_video.tweet:1]:
```suggestion
For everyone who missed or wants to revisit Luca's talk about #JuliaLang package development of our #ResearchSoftwareSeminarSeries: 
```

**Review by bast — APPROVED** [review]
Awesome. Thanks again for patience with this. I think Luca should then pull the trigger to merge once it's good time.

**Comment by lucaferranti** [comment]
The youtube video atm doesn't have a thumbnail. I don't know if that's something fixable (manually add a picture as thumbnail?) If it is, maybe this could be merged after that?

**Comment by lucaferranti** [comment]
It seems the video now has a thumbnail: https://www.youtube.com/results?search_query=package+development+in+julia so I think  I can "pull the trigger" @bast is this ready to go?

**Comment by bast** [comment]
In my opinion ready to go. But I wanted you to push the button since it's about your video. Thanks again.

**Pull request merged**